test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetUITestCase.cs

```diff
@@ -156,6 +158,318 @@ public void UpdatePackageFromUI()
             CommonUtility.AssertPackageInPackagesConfig(VisualStudio, project, "newtonsoft.json", "10.0.3", XunitLogger);
         }
 
+        [StaFact]
+        public async Task InstallPackageFromUI_PackageNamespace_WithSingleFeed_Match_Succeeds()
```

Comment:
This name is a lot better than the ones in NuGetPackageManagerTests!

Reply:
Ok. Improved names in `NuGetPackageManagerTests.cs`.